AzureCalling/Controllers/IntroViewController.swift

```diff
@@ -0,0 +1,262 @@
+    private var topBar: UIView!
+    private var userAvatar: MSFAvatar!
+    private var userDisplayName: FluentUI.Label!
+
+    // MARK: ViewController Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        createControls()
+        layoutView()
+
+        handleAuthState()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: animated)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        self.navigationController?.setNavigationBarHidden(false, animated: animated)
+    }
+
+    // MARK: UI layout
+    private func createControls() {
+        signinButton = FluentUI.Button.createWith(
+            style: .primaryFilled,
+            title: "Sign in",
+            action: { [weak self] _ in self?.loginAAD() }
+        )
+        startCallButton = FluentUI.Button.createWith(
+            style: .primaryFilled,
+            title: "Start a call",
+            action: { [weak self] _ in self?.startCall() }
+        )
+        joinCallButton = FluentUI.Button.createWith(
+            style: .primaryOutline,
+            title: "Join a call",
+            action: { [weak self] _ in self?.joinCall() }
+        )
+        signOutButton = FluentUI.Button.createWith(
+            style: .borderless,
+            title: "Sign out",
+            action: { [weak self] _ in self?.showSignOutAlert() }
+        )
+    }
+
+    private func layoutView() {
+        view.backgroundColor = ThemeColor.lightSurfacesSecondary
+        layoutButtons()
+        layoutMainContainer()
+        layoutTopBar()
+    }
+
+    private func layoutButtons() {
+        let stackView = UIStackView(
+            arrangedSubviews: [
+                signinButton,
+                startCallButton,
+                joinCallButton
+            ]
+        )
+        stackView.axis = .vertical
+        stackView.spacing = 8
+        view.addSubview(stackView)
+        signinButton.expandHorizontallyInSuperView()
+        startCallButton.expandHorizontallyInSuperView()
+        joinCallButton.expandHorizontallyInSuperView()
+        stackView.expandHorizontallyInSuperView(withEqualMargin: 16)
+        stackView.pinToBottom()
+    }
+
+    private func layoutMainContainer() {
+        let titleLabel = FluentUI.Label(style: .title1)
+        titleLabel.text = "Video calling sample"
+
+        let builtWithLabel = FluentUI.Label(style: .body)
+        builtWithLabel.text = "Built with"
+        builtWithLabel.directionalLayoutMargins.top = 24
+
+        let acsImageLabel = FluentUI.Label(style: .button1)
+        acsImageLabel.text = "Azure Communication Services"
+        acsImageLabel.textColor = ThemeColor.primary
+
+        let acsLabelStack = UIStackView(arrangedSubviews: [
+            UIImageView(image: UIImage(named: "acsLogo")),
```

Comment:
I think we're missing "acsLogo" image in Assets.xcassets which causes the image not being displayed.

Reply:
I am contacting Alex about getting it.